### PR TITLE
Enable keeping the DateRangePicker open after the user selects their dates

### PR DESCRIPTION
**Pull request by hen-zone**
Just passes the `keepOpenOnDateSelect` prop through to the `react-dates` component.

Required to satisfy design review.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "he-react-ui",
-  "version": "0.0.87",
+  "version": "0.1.0",
```
Comment (xow) [inline package.json]: Are we ready for this @BobHayden

**Comment by xow** [comment]
From a UX perspective this feels much nicer

File: package.json (hunk 1 above)
Comment (xow) [reply package.json]: Let's make the jump

**Comment by xow** [comment]
I was too slow, there's now conflicts :cry: this one at a time thing is going to get annoying.

But since we have the allComponents test we always need to have PRs up to date

**Comment by hen-zone** [comment]
I'll resolve the conflicts now. Perhaps the allComponents test is more trouble than it's worth; If we just have regular tests for every feature then we don't need it anyway. It's useful to catch new components/fixtures without an explicit test, but actually coveralls will be catching those because the coverage will decrease. Your thoughts on removing it for now?

**Pull request merged**